signer/src/stacks/contracts.rs

```diff
@@ -46,15 +49,40 @@ pub struct StacksTxPostConditions {
     pub post_conditions: Vec<TransactionPostCondition>,
 }
 
-/// A trait to ease construction of a StacksTransaction making sBTC related contract calls.
+/// A trait to ease construction of a StacksTransaction making sBTC related
+/// contract calls.
 pub trait AsContractCall {
-    /// Converts this struct to a Stacks contract call. The deployer is the
+    /// The name of the clarity smart contract that relates to this struct.
+    fn contract_name() -> &'static str;
+    /// The specific function name that relates to this struct.
+    fn function_name() -> &'static str;
```

Comment:
These could be constants, as you previously had in the `AcceptWithdrawalV1` struct
```suggestion
    /// The name of the clarity smart contract that relates to this struct.
    const CONTRACT_NAME: &'static str;
    /// The specific function name that relates to this struct.
    const FUNCTION_NAME: 'static str;
```

Reply:
Oh for some reason I didn't think we could do that